net-wireless/wavemon/files/wavemon-0.9.3-build.patch

```diff
@@ -0,0 +1,40 @@
+--- a/configure.ac	2020-12-26 17:44:01.616731241 +0100
++++ b/configure.ac	2020-12-26 17:45:02.447292151 +0100
+@@ -22,6 +22,8 @@
```

Comment:
We like to have the headers from the original source if you found this somewhere, and if not, still in git-patch format.

Reply:
I re-created the patch to adapt it to this new version, so I guess that the headers reflect the original source?